lib/widgets/modular_widgets/wallet_connect_widgets/wallet_connect_session_list_card.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:walletconnect_flutter_v2/walletconnect_flutter_v2.dart';
+import 'package:zenon_syrius_wallet_flutter/blocs/wallet_connect/wallet_connect_sessions_bloc.dart';
+import 'package:zenon_syrius_wallet_flutter/main.dart';
+import 'package:zenon_syrius_wallet_flutter/utils/extensions.dart';
+import 'package:zenon_syrius_wallet_flutter/widgets/widgets.dart';
+
+const String _kWidgetTitle = 'WalletConnect Sessions List';
+const String _kWidgetDescription =
+    'A session refers to a live connection between a user\'s wallet and a dApp '
+    'A session allows the dApp to communicate with the wallet '
+    'securely over the Internet';
+
+class WalletConnectSessionsCard extends StatefulWidget {
+  const WalletConnectSessionsCard({Key? key}) : super(key: key);
+
+  @override
+  State<WalletConnectSessionsCard> createState() =>
+      _WalletConnectSessionsCardState();
+}
+
+class _WalletConnectSessionsCardState extends State<WalletConnectSessionsCard> {
+  final WalletConnectSessionsBloc _sessionsBloc =
+      sl.get<WalletConnectSessionsBloc>();
+
+  @override
+  Widget build(BuildContext context) {
+    return CardScaffold(
+      title: _kWidgetTitle,
+      description: _kWidgetDescription,
+      childBuilder: () => _getCardBody(),
+      onRefreshPressed: () => _sessionsBloc.refreshResults(),
+    );
+  }
+
+  Widget _getCardBody() {
+    return _buildPairingsTable();
+  }
+
+  Widget _buildPairingsTable() {
+    return InfiniteScrollTable<SessionData>(
+      disposeBloc: false,
+      bloc: _sessionsBloc,
+      headerColumns: const [
```

Comment:
Should a "deactive session" button (similar to what the pairing list has) be added so that users can disconnect their sessions? Now the user can only disconnect the entire pairing and this means the dapp will have to show the WallectConnect popup again when the user wants to start a new session.